Containers/talk-recording/Dockerfile

```diff
@@ -28,13 +29,14 @@ RUN set -ex; \
     python3 -m pip install /src/recording/src; \
     rm -rf /src; \
     chown recording:recording -R \
-        /tmp; \
+        /tmp /etc/recording.conf; \
```

Comment:
maybe instead of copying the file as step do a touch to create the file and them run chown?

Reply:
Sounds good!

Reply:
can you please do it?

Reply:
Done with https://github.com/nextcloud/all-in-one/commit/a797f624ec1d0456dd80f99bc3825aab2ffadd77